update_command.php

```diff
@@ -4,6 +4,8 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Command\Command;
 
+use function Symfony\Component\String\u;
```

Comment:
What's this here for? I can't find it being used?

Reply:
Typo, removed